usaspending_api/etl/tests/integration/test_update_file_c_linkages_in_delta.py

```diff
@@ -17,7 +17,7 @@
 common_faba_info = {"submission_id": "0", "distinct_award_key": "fake_key"}
 
 award_records = [
-    {"id": 1, "expected_update": False},
+    {"id": 1, "expected_update": True},
```

Comment:
This Award was already being used by a few other test cases, where it is important to ensure that it's not being updated. Can you define a 12th award to use for your test case.

Reply:
Same for FABA ID 5. Can you create a new FABA record for your test case?

Reply:
This is one of the consequences of having a test, test multiple things.

Reply:
I'll add an id 12